Package.swift

```diff
@@ -23,5 +23,5 @@ let package = Package(
       ]
     )
   ],
-  swiftLanguageVersions: [4]
+  swiftLanguageVersions: [.v4_2, .version("5")]
```

Comment:
Shouldn't we migrate to `// swift-tools-version:5.0` and thus be able to use `.v5` here?

Reply:
@djbe why did you mark this as resolved? Forgot the rationale for not using `// swift-tools-version:5.0` given we're now requiring Swift 5 to build anyway? Or do we still want to be Swift-4.2-compatible (tho if so, we don't check about this on CI, so not sure if we can guarantee such support officially 🤔 )

Reply:
Note the other `Package@swift5.swift` file.

Reply:
Good remark about testing swift 4.2, lemme see if I can create an image for it.

Reply:
Ah right.

I think we could consider merging this PR, and open a separate one to:

- Either drop Swift 4.2 support (removing `Package.swift` and renaming `Package@swift5.swift` to `Package.swift`)
- or keep supporting it by adding a Docker image for it and the relevant entry in CI config

Reply:
Sounds good 👍